lib/adapters/tool/index.ts

```diff
@@ -31,6 +31,7 @@ export interface ToolAdapter {
     initGuiApi(): void;
     readTests(paths: string[], cliTool: CommanderStatic): Promise<TestCollectionAdapter>;
     run(testCollection: TestCollectionAdapter, tests: TestSpec[], cliTool: CommanderStatic): Promise<boolean>;
+    runWithoutRetries(testCollection: TestCollectionAdapter, tests: TestSpec[], cliTool: CommanderStatic): Promise<boolean>;
```

Comment:
Add new method in order to not forget in new adapters to implement logic with disable retries before run tests